README.md

```diff
@@ -1,65 +1,126 @@
 
-Live Link
-Live Link
+## Built With
 
-Built With
-Ruby
-Ruby on Rails
-Getting Started
-To get a local copy up and running follow these simple example steps.
+- Ruby
+- Rails
+- Postgresql
+- Linters
 
-git clone git@github.com:Ambrosegithub/Budgetary-App.git
+### Key Features
 
-cd budget-app
+- [Log in and registration of the user]
+- [Category list , Transaction list]
+- [Functionality to Generate a Expenses for our categories with total cost]
 
-Prerequisites
-Ruby Rails PostgresSQL
+## Video Demo
+[Video Link](https://www.loom.com/share/713ff3bc31f4405ab59df336b9e75c7c)
 
-Setup
-Install gems with:
+# Live Presentation link
+[Live Link](https://gentle-spire-11113.herokuapp.com/)
 
-bundle install
-Setup database with:
+## Getting Started
+-To get a local copy up and running follow these simple example steps.
 
-rails db:create
-rails db:migrate
-Start server with:
+### Prerequisites
 
-rails server
-Open http://localhost:3000/ in your browser.
+- A Command Line Interface.
+- Having Ruby installed.
+- Setup Rails
+- Postgresql
+- A code editor like Visual Studio Clone.
+- Git installed.
+- 
 
-Run tests
-Install npm with:
+To check if you have Git, Postgres and Ruby installed you can use the commands below:
 
-npm i
-Install rspec with:
+ ```sh
+   $ git --version
+   $ ruby --version && irb
+   ```
+If you don't have Ruby, you can find a complete list of ways to install Ruby, including using third-party tools like rvm, see:
 
-bundle install
-and
+(https://www.ruby-lang.org/en/downloads/)
+### 🛠️ Installation & Set Up
 
-rails generate rspec:install
-run the test with:
+If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.
 
-rspec spec
-Author
-👤 Isigba Ambrose
 
-GitHub: @Ambrosegithub
-Twitter: @Glorious12
-🤝 Contributing
+- Open the [Repository Link](https://github.com/Ambrosegithub/Budgetary-App) of this project in GitHub.
+
+- Once the files are on your machine, open capstone-ruby folder in [Visual Studio Code](https://code.visualstudio.com/).
+
+### Install
+ Start by cloning the repository.
+ 
+- git@github.com:Ambrosegithub/Budgetary-App.git
+
+  Navigate to the created folder.
+
+- Initialize your project with Git.
+- Make sure that your project has Postgres database set up.
+- Run rails server and visi thttp://127.0.0.1:3000/ in your browser!
+
+  ### run the following in your terminal:
+  - gem install rails
+  - gem install pg
+  - bundle install (use this whenever you want add a gem in your gem file)
+  - rails db:create
+  - change your database password to suit yours in database.yml
+  
+  
+
+### Usage
+--Once you have your repository cloned on your machine, feel free to modify it as needed.
+- To add your own page
+
+### Testing with Rspec and Capybara
+  - In your terminal run > 
+     bundle exec rspec
+  - run rspec spec
+
```

Comment:
- [ ] OPTIONAL
Kindly ensure to add the _Future Features_ section to your ReadMe, which should include 1-3 potential features to be added